app/jobs/appointment_notification/worker.rb

```diff
@@ -41,22 +41,25 @@ def send_message(notification, communication_type)
       NotificationService.new
     end
 
-    if communication_type == "missed_visit_whatsapp_reminder"
+    case communication_type
+    when "whatsapp"
```

Comment:
Should we keep the old types around for a while (`when "whatsapp", "missed_visit_whatsapp_reminder"`) to handle any jobs that are executed before the migration switches everyone's comms type? It would be a "transitional backwards compatibility", similar to the communication types hash below - https://github.com/simpledotorg/simple-server/pull/2613/files#diff-8bef04c7770083f39d473b82194bccd100e857238d53212f8a79d72731dbdcebR12-R14

Reply:
The reason I kept the types in the enum is because I want the records to continue to be valid and update-able. I didn't keep them here because there shouldn't be a case where we want to send any of the notifications that currently exist. However... I've been wrong many times before, and it's probably not a bad idea to keep there here to be safe. I'll re-add and add a note in the migration ticket to remove the references.

Reply:
Yeah let's leave the backwards compatibility in here for a day, and then rip it out.